guardrails/classes/history/call.py

```diff
@@ -72,66 +71,42 @@ def __init__(
         self.exception = exception
 
-    @property
-    def prompt(self) -> Optional[str]:
-        """The prompt as provided by the user when initializing or calling the
-        Guard."""
-        return self.inputs.prompt
-
     @property
     def prompt_params(self) -> Optional[Dict]:
         """The prompt parameters as provided by the user when initializing or
         calling the Guard."""
         return self.inputs.prompt_params
 
     @property
-    def compiled_prompt(self) -> Optional[str]:
-        """The initial compiled prompt that was passed to the LLM on the first
-        call."""
-        if self.iterations.empty():
-            return None
-        initial_inputs = self.iterations.first.inputs  # type: ignore
-        prompt: Prompt = initial_inputs.prompt  # type: ignore
-        prompt_params = initial_inputs.prompt_params or {}
-        if initial_inputs.prompt is not None:
-            return prompt.format(**prompt_params).source
-
-    @property
-    def reask_prompts(self) -> Stack[Optional[str]]:
-        """The compiled prompts used during reasks.
-
-        Does not include the initial prompt.
-        """
-        if self.iterations.length > 0:
-            reasks = self.iterations.copy()
-            initial_prompt = reasks.first
-            reasks.remove(initial_prompt)  # type: ignore
-            return Stack(
-                *[
-                    r.inputs.prompt.source if r.inputs.prompt is not None else None
-                    for r in reasks
-                ]
-            )
-
-        return Stack()
+    def messages(self) -> Optional[Union[Messages, list[dict[str, str]]]]:
+        """The messages as provided by the user when initializing or calling
+        the Guard."""
+        return self.inputs.messages
 
     @property
-    def instructions(self) -> Optional[str]:
-        """The instructions as provided by the user when initializing or
-        calling the Guard."""
-        return self.inputs.instructions
-
-    @property
-    def compiled_instructions(self) -> Optional[str]:
-        """The initial compiled instructions that were passed to the LLM on the
+    def compiled_messages(self) -> Optional[list[dict[str, str]]]:
+        """The initial compiled messages that were passed to the LLM on the
         first call."""
         if self.iterations.empty():
             return None
         initial_inputs = self.iterations.first.inputs  # type: ignore
-        instructions: Instructions = initial_inputs.instructions  # type: ignore
+        messages = initial_inputs.messages
         prompt_params = initial_inputs.prompt_params or {}
-        if instructions is not None:
-            return instructions.format(**prompt_params).source
+        compiled_messages = []
+        if messages is None:
+            return None
+        for message in messages:
+            content = message["content"].format(**prompt_params)
+            if isinstance(content, (Prompt, Instructions)):
```

Comment:
It looks like Prompt was removed from the imports.  Does this still run?  Was a different prompt imported?